akka-remote/src/test/scala/akka/remote/classic/ActorsLeakSpec.scala

```diff
@@ -203,15 +203,13 @@ class ActorsLeakSpec extends AkkaSpec(ActorsLeakSpec.config) with ImplicitSender
         // All system messages has been acked now on this side
 
         // This will make sure that no SHUTDOWN message gets through
-        Await.ready(RARP(system).provider.transport.managementCommand(ForceDisassociate(remoteAddress)), 3.seconds)
+        Await.result(RARP(system).provider.transport.managementCommand(ForceDisassociate(remoteAddress)), 3.seconds)
 
       } finally {
         remoteSystem.terminate()
       }
 
-      EventFilter.warning(pattern = "Association with remote system", occurrences = 1).intercept {
-        Await.ready(remoteSystem.whenTerminated, 10.seconds)
-      }
+      Await.result(remoteSystem.whenTerminated, 10.seconds)
 
       EventFilter[TimeoutException](occurrences = 1).intercept {}
```

Comment:
this is somewhat weird, but I guess that's how to do it with the EventFilter testkit